SoftFox/SoftFox/SoftFox.cpp

```diff
@@ -5,119 +5,138 @@
 			{
+				//Create a case for quitting the window and set running to false to deconstruct the window
+				//Break or system breaks
+				case SDL_QUIT:
+					running = false;
+					break;
+
+				default:
+					break;
+			}
+		}
 
-				//Set a boolean to keep the window running until false
-				bool running = true;
-				int playerX = WINDOW_WIDTH / 2;
-				int playerY = WINDOW_HEIGHT / 2;
-
-				int platformX = 40;
-				int platformY = 400;
-
-				while (running)
-				{
-					//Set an event
-					SDL_Event ev;
-					if (SDL_PollEvent(&ev))
-					{
-						switch (ev.type)
-						{
-							//Create a case for quitting the window and set running to false to deconstruct the window
-							//Break or system breaks
-						case SDL_QUIT:
-							running = false;
-							break;
-
-						default:
-							break;
-						}
-					}
-
-					// Check keyboard state
-					const Uint8* keyboardState = SDL_GetKeyboardState(nullptr);
-					if (keyboardState[SDL_SCANCODE_UP])
-						playerY -= PLAYER_MOVEMENT_SPEED;
-					if (keyboardState[SDL_SCANCODE_DOWN])
-						playerY += PLAYER_MOVEMENT_SPEED;
-					if (keyboardState[SDL_SCANCODE_LEFT])
-						playerX -= PLAYER_MOVEMENT_SPEED;
-					if (keyboardState[SDL_SCANCODE_RIGHT])
-						playerX += PLAYER_MOVEMENT_SPEED;
-
-					//Change the colour of the background renderer and then clear the colour
-					SDL_SetRenderDrawColor(renderer, 0, 0, 0, 255);
-					SDL_RenderClear(renderer);
+		// Check keyboard state
+		const Uint8* keyboardState = SDL_GetKeyboardState(nullptr);
+		if (keyboardState[SDL_SCANCODE_UP])
+			playerY -= PLAYER_MOVEMENT_SPEED;
+		if (keyboardState[SDL_SCANCODE_DOWN])
+			playerY += PLAYER_MOVEMENT_SPEED;
+		if (keyboardState[SDL_SCANCODE_LEFT])
+			playerX -= PLAYER_MOVEMENT_SPEED;
+		if (keyboardState[SDL_SCANCODE_RIGHT])
+			playerX += PLAYER_MOVEMENT_SPEED;
+
+		//Change the colour of the background renderer and then clear the colour
+		SDL_SetRenderDrawColor(renderer, 0, 0, 0, 255);
+		SDL_RenderClear(renderer);
+
+		drawLevel();
 					
-					//Drawing player sprite (texture class)
-					playerSprite->render(renderer, playerX, playerY, SPRITE_SIZE, SPRITE_SIZE);
+		//Drawing player sprite (texture class)
+		//playerSprite.render(renderer, playerX, playerY, SPRITE_SIZE, SPRITE_SIZE);
 					
-					//Draw platform sprite texture
-					platformSprite->render(renderer, 0, platformY, WINDOW_WIDTH*2, 10);
+		//Draw platform sprite texture
+		//platformSprite.render(renderer, 0, platformY, WINDOW_WIDTH*2, 10);
 					
-					SDL_RenderPresent(renderer);
-					
-				}
+		SDL_RenderPresent(renderer);				
+	}
+}
 
-				//SDL_Delay(2000);
+void SoftFox::drawTile(int x, int y, SDL_Texture* texture)
+{
+	SDL_Rect dest;
+	dest.x = x* tileSize;
+	dest.y = y* tileSize;
+	dest.w = tileSize;
+	dest.h = tileSize;
+	SDL_RenderCopy(renderer, texture, nullptr, &dest);
+}
 
-				//Remove the renderer
-				SDL_DestroyRenderer(renderer);
+void SoftFox::drawLevel()
+{
+	for (int y = 0; y < level->getHeight; y++)
```

Comment:
`level->getHeight` is a function, but you've missed off the `()` to make it a function call. Same with `getWidth` below.